pkg/api/handlers/credentials.go

```diff
@@ -51,3 +51,12 @@ func convertCredential(cred gptscript.Credential) types.Credential {
 		ExpiresAt: types.NewTimeFromPointer(cred.ExpiresAt),
 	}
 }
+
+func RecreateAllCredentials(req api.Context) error {
+	err := req.GPTClient.RecreateAllCredentials(req.Context())
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
micro-nit:
```suggestion
	return req.GPTClient.RecreateAllCredentials(req.Context())
```

Reply:
I'm just gonna blame AI for this one lol